### PR TITLE
Fix macos build

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes https://github.com/betaflight/betaflight-configurator/pull/3657 build

### CODE REVIEW

**Comment by nerdCopter** [comment]
if it works, may or may not be temporary.
on external project i'm moving to `gulp-macdmg` as it removes `gulp-util` and maybe `macos-alias` (unclear).
package.json's `optionaldependencies` not working alone, `npm install --python=python2.7` not working alone. unfortunate requiring both to work, which is silly.

**Review by nerdCopter — APPROVED** [review]

**Comment by haslinghuis** [comment]
Removing 2nd commit

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Pull request merged**
